src/pages/single-video/SingleVideoPage.js

```diff
@@ -94,103 +98,148 @@ const SingleVideoPage = props => {
+
+      if (!error)
+        setCurrentVideo(prev => ({
+          ...prev,
+          comments: prev.comments.concat(commentData),
+        }));
+      isReadyToComment = true;
+    }
+
+    commentInputRef.current.value = '';
+
+    setEnableComment(false);
+  };
+
   return (
     <Fragment>
       <main className="main-single-video-page">
         {currentVideo && (
-          <div className="video-container">
-            <iframe
-              width="100%"
-              height="100%"
-              src={`https://www.youtube.com/embed/${currentVideo.videoId}?autoplay=1`}
-              title="YouTube video player"
-              frameborder="0"
-              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-              allowfullscreen
-            ></iframe>
-          </div>
-        )}
-        <div className="suggestions-container">
-          {videos.map(video => (
-            <VideoCard video={video} wantWatchLaterButton={true} />
-          ))}
-        </div>
-        <div className="video-details">
-          <h3>{currentVideo?.title}</h3>
-          <div className="flex space-between">
-            <div className="flex gap text-small">
-              <p>1,00,000 views</p>
-              <p>{dateFormatter(currentVideo.publishedAt)}</p>
+          <div>
+            <div className="video-container">
+              <iframe
+                width="100%"
+                height="100%"
+                src={`https://www.youtube.com/embed/${currentVideo.videoId}?autoplay=1`}
+                title="YouTube video player"
+                frameBorder="0"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                allowFullScreen
+              ></iframe>
             </div>
-            <div className="video-actions flex gap ">
-              <input
-                onChange={likedDislikedHandler.bind(null, 'like')}
-                type="checkbox"
-                id="liked"
-                checked={currentVideo.liked?.includes(userId)}
-              />
-              <label htmlFor="liked">
-                {currentVideo.liked?.includes(userId) ? (
-                  <i className="fas fa-thumbs-up"></i>
-                ) : (
-                  <i className="far fa-thumbs-up"></i>
-                )}
-                {` ${numberFormatter(currentVideo.liked?.length)}`}
-              </label>
-              <input
-                onChange={likedDislikedHandler.bind(null, 'dislike')}
-                type="checkbox"
-                id="disliked"
-                checked={currentVideo.disliked?.includes(userId)}
-              />
-              <label htmlFor="disliked">
-                {currentVideo.disliked?.includes(userId) ? (
-                  <i className="fas fa-thumbs-down"></i>
-                ) : (
-                  <i className="far fa-thumbs-down"></i>
+
+            <div className="video-details">
```

Comment:
Suggestion: instead of assigning value this way you can use object destructuring.